content/en/docs/reference/instrumentation/metrics.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Kubernetes Metrics Across Components
+content_type: instrumentation
+---
+
+
+## Metrics
+
+These are the metrics which are exported in Kubernetes components (i.e. kube-apiserver, scheduler, kube-controller-manager, kube-proxy, cloud-controller-manager). 
+
+(auto-generated 2022 Oct 25)
+
+### List of Kubernetes Metrics
+
+<table class="table">
+<thead>
+    <tr>
+        <td width="20%">Name</td>
+        <td width="10%">Stability Level</td>
+        <td width="10%">Type</td>
+        <td width="30%">Help</td>
+        <td width="20%">Labels</td>
+        <td width="10%">Const Labels</td>
```

Comment:
- please use `<th>` elements in this row.
- please give each column a class, to allow CSS styling
- please _don't_ set a width (use SCSS instead)

Reply:
Future work: also detail which components export each metric.

Reply:
Yeah can't do this yet, unfortunately, since it's actually not always accessible from the metric definition. But it is on my list of things I want.

Reply:
Where do I place the scss?

Reply:
The SCSS goes in `assets/scss`